components/datetime/src/skeleton/mod.rs

```diff
@@ -42,20 +42,18 @@ mod test {
         DataPayload<DateSkeletonPatternsV1Marker>,
     ) {
         let provider = icu_testdata::get_provider();
-        let locale: Locale = "en-u-ca-gregory".parse().unwrap();
+        let locale = "en-u-ca-gregory".parse::<Locale>().unwrap();
```

Comment:
`question`: Can we already use a macro for that locale?

Reply:
nope